### DISCUSSION TITLE
handbook/0.1/posts/20221214_boas_praticas_criacao_posts_dcta/

### DISCUSSION THREAD

**Discussion by giscus** [transparencia-mg/handbook#28] [Announcements]
# handbook/0.1/posts/20221214_boas_praticas_criacao_posts_dcta/

https://transparencia-mg.github.io/handbook/0.1/posts/20221214_boas_praticas_criacao_posts_dcta/

**Comment by gabrielbdornas** [comment]
@Andrelamor, escrevo este para testar nosso novo [comment system](https://squidfunk.github.io/mkdocs-material/setup/adding-a-comment-system/?h=com). 

Aproveito a oportunidade para te pedir ajuda para complementar este post.

**Reply by gabrielbdornas** [reply]
Testando resposta de @gabrielbdornas

**Comment by Andrelamor** [comment]
teste
